bitname-protocol/bitname-cli
# Add regtest support to the client

Closes #54 

This will make integration testing way, way easier, even manually.

* Remove the old `--testnet` cli flag
* Add a new `--network` flag, which takes either main, testnet, or regtest
* Add a new pubkey prefix for regtest
* Adds new tests to keep coverage high
* Add a Docker Compose config file to easily run bitcoind and ElectrumX in regtest mode
* Adds some scripts to easily interact with the regtest chain
* Adds a guide on how to use the new regtest utils

It would be great if someone/multiple people could follow the regtest guide I included to make sure that all works/is unambiguous, as well as running a `bitname` flow in regtest mode.
